examples/usecases/ecommerce-session-based-next-item-prediction-for-fashion.ipynb

```diff
@@ -70,7 +70,7 @@
     "\n",
```

Comment:
<p>Can we not select by Tag sequence? instead of schema_model.select_by_name(</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;['item_id_list_seq', 'f_47_list_seq', 'f_68_list_seq']</p><p> </p><p>Similar, do we have a tag for schema_model.select_by_name(</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;['item_id_last']</p><p>&nbsp;&nbsp;&nbsp;&nbsp;) ?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/849/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='88'/>